.gitconfig

```diff
@@ -1,17 +1,34 @@
 [user]
-	name = Ta-Seen Islam
```

Comment:
This was done to synchronise the author property of my commits and GitHub's automated commits (e.g. from merging Pull Requests).